packages/stream-tile/src/tile-document.ts

```diff
@@ -52,6 +53,16 @@ export interface TileMetadataArgs {
    * identical documents will generate unique StreamIDs and be able to be updated independently.
    */
   deterministic?: boolean
+
+  /**
+   * If true, all changes to the 'controllers' array are disallowed.  This guarantees that the
+   * Stream will always have the same controller. Especially useful for Streams controlled by
+   * DIDs that can have ownership changes within the DID itself, such as did:nft, as setting this
+   * would prevent the current holder of the NFT from changing the Stream's controller to that
+   * user's personal DID, which would prevent the ownership of the Stream from changing the next
+   * time the NFT is traded.
+   */
+  forbidControllerChange?: boolean
```

Comment:
Could we call it something more concise, like `nonTransferable`?

Reply:
I'd be okay with calling it `nonTransferable` in the commit header, but I think the user-facing argument should be more descriptive.  It doesn't seem to me like it would be immediately clear to users what `nonTransferable` actually means

Reply:
ugh, actually having the argument have a different name that the value that shows up in the state object is likely to just cause more confusion I think.  Probably makes sense to keep the names the same wherever possible.

Given that, I still prefer `forbidControllerChange` to `nonTransferable`.  For one thing, if the controller is a did:nft, the stream still *is* transferable, even with this property set, which seems confusing.

I'm not thrilled with the name `forbidControllerChange`, but it's the best I've been able to come up with, and at least it's pretty descriptive of what it actually does